src/aap_eda/api/views/organization.py

```diff
@@ -123,15 +124,24 @@ def destroy(self, request, *args, **kwargs):
         ],
     )
     @action(
-        detail=True,
+        detail=False,
```

Comment:
why do we need to turn it to False?

Reply:
`detail=False` is for when we are dealing with a collection of instances (teams) rather than a single instance; additionally, it also helps us use Team specific queryset and filterset on organization viewset. This follows the pattern that we have for existing endpoints, such as `activations/<id>/instances/`.